src/Strategy/NullCacheStrategy.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Kevinrob\GuzzleCache\Strategy;
+
+use Kevinrob\GuzzleCache\CacheEntry;
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ResponseInterface;
+
+class NullCacheStrategy implements CacheStrategyInterface
+{
+
+    /**
+     * @inheritDoc
+     */
+    public function fetch(RequestInterface $request)
+    {
+        return null;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function cache(RequestInterface $request, ResponseInterface $response)
+    {
+        return true;
```

Comment:
Hm, when I read PHPdoc's return type hint for `CacheStrategyInterface::cache()` and  `CacheStrategyInterface::update()`, it's:
```php
/**
* @return bool true if success
*/
```

So I assumed returning `false` would mean a failure. However, to my eyes, nothing failed: the `NullCacheStrategy` accomplished its job with success (i.e. doing nothing). 

I feared a further implementation would do this:
```php
if (false === $strategy->cache($request, $response) {
    throw new \RuntimeException("There was an error when trying to cache the request/response");
}
```

Then, the `NullCacheStrategy` would systematically fail.

What's your opinion?